Bindings/Java/Matlab/Utilities/osimC3D.m

```diff
@@ -1,30 +1,67 @@
 classdef osimC3D < matlab.mixin.SetGet
-% osimC3D C3D data into OpenSim Tables
-%   Utility  Class that uses OpenSim's C3DFileAdapter() class to produce
-%   OpenSim tables. Methods include Simbody rotation.
+% osimC3D(filepath, ForceLocation)
+%   C3D data to OpenSim Tables.
+%   OpenSim Utility Class
+%   Inputs:
+%   filepath           Full path to the location of the C3D file
+%   ForceLocation      Integer value for representation of force from plate
+%                      0 = forceplate orgin, 1 = surface, 2 = COP
```

Comment:
What do you mean by `surface`; does `surface` mean "PointOfWrenchApplication"? Based on the code below, isn't COP denoted by `1`?